packages/php-wasm/compile/Dockerfile

```diff
@@ -1055,5 +1069,13 @@ RUN \
         rm /root/output/php.js && \
         \
         # Rename the build files to their final names
-        mv /root/output/php-module.js "/root/output/$JS_FILENAME" && \
-        mv /root/output/php.wasm "/root/output/$WASM_FILENAME"
\ No newline at end of file
+        mv /root/output/php-module.js "/root/output/$JS_FILENAME"; \
+        # mv /root/output/php.wasm "/root/output/$WASM_FILENAME";
+        mkdir -p /root/output/${PHP_VERSION_ESCAPED}/; \
```

Comment:
Let's stick with `&&` in this and next lines until the Dockerfile uses pipefail by default

Reply:
Same idea: https://github.com/WordPress/wordpress-playground/pull/639#discussion_r1346673235